### PR TITLE
Remove `org.apache.commons:commons-lang3` and sort entries in pom.xml

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,13 +6,13 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>146</version>
+        <version>147</version>
```
Comment (mosabua) [inline pom.xml:9]: Please mention this update in the a commit message or do it in a separate commit

**Review by mosabua — APPROVED** [review]
One minor nit .. but essentially good to go.

File: pom.xml (hunk 1 above)
Comment (ebyhr) [reply pom.xml:9]: The commit body already mentions: 
```
Also, upgrade airbase to 147 that contains sortpom-maven-plugin.
```
